src/test/resources/IBMdefined/RecordSeparatedFieldFormat.xsd

```diff
@@ -0,0 +1,56 @@
+	* and other DFDL constructs to assist in the creation of DFDL schemas that model
+	* text data format that consists of a number of records or segments having an optional
+	* header, repeating body and optional trailer. The record fields have a variable length
+	* and are separated by a delimiter.  
+	* 
+	* 
+	* Example use: 
+	* 
+	*      <xsd:import namespace="http://www.ibm.com/dfdl/RecordSeparatedFieldFormat" schemaLocation="RecordSeparatedFieldFormat.xsd"/>
+	******************************************************************* -->
+	
+<schema
+	targetNamespace="http://www.ibm.com/dfdl/RecordSeparatedFieldFormat"
+	xmlns="http://www.w3.org/2001/XMLSchema"
+	xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+	xmlns:tns="http://www.ibm.com/dfdl/RecordSeparatedFieldFormat">
+	<annotation>
+		<appinfo source="http://www.ogf.org/dfdl/">
+
+			<dfdl:defineEscapeScheme name="RecordEscapeScheme">
+				<dfdl:escapeScheme escapeKind="escapeBlock" escapeBlockStart='"' escapeBlockEnd='"' escapeCharacter='"'
+					escapeEscapeCharacter='"' extraEscapedCharacters='%#x0D; %#x0A;' generateEscapeBlock="whenNeeded"></dfdl:escapeScheme>
+			</dfdl:defineEscapeScheme>
+
+
+			<dfdl:defineFormat name="RecordSeparatedFieldsFormat">
+				<dfdl:format encoding="UTF-8" ignoreCase="no" byteOrder="bigEndian" representation="text" textPadKind="none"
+					textStringJustification="left" truncateSpecifiedLengthString="no" decimalSigned="yes" textNumberCheckPolicy="lax"
+					textNumberJustification="right" textZonedSignStyle="asciiStandard" textBooleanJustification="left" sequenceKind="ordered"
+					choiceLengthKind="implicit" separator="," separatorPolicy="suppressed" separatorPosition="infix"
+					documentFinalTerminatorCanBeMissing="no" lengthUnits="characters" lengthKind="delimited" textTrimKind="padChar"
+					textStandardBase="10" textStandardDecimalSeparator="." textStandardExponentRep="E" alignment="1" alignmentUnits="bytes" leadingSkip="0" trailingSkip="0" fillByte="0" textStringPadCharacter="%SP;"
+					initiatedContent="no" textNumberPadCharacter="%SP;" textBooleanPadCharacter="%SP;" textNumberRep="standard"
+					textStandardGroupingSeparator="," textStandardZeroRep="" textNumberRoundingMode="roundUp" nilValueDelimiterPolicy="initiator"
+					emptyValueDelimiterPolicy="initiator" escapeSchemeRef="tns:RecordEscapeScheme" textStandardInfinityRep="Inf" textStandardNaNRep="NaN" textNumberRounding="pattern" calendarPatternKind="implicit" calendarPattern="yyyy-MM-dd'T'HH:mm:ss" calendarCheckPolicy="lax" calendarTimeZone="UTC" calendarObserveDST="yes" calendarFirstDayOfWeek="Monday" calendarDaysInFirstWeek="4" calendarCenturyStart="53" calendarLanguage="en-US" textCalendarJustification="left" textBooleanTrueRep="true" textBooleanFalseRep="false" textCalendarPadCharacter="%SP;" 
+					initiator="" terminator="" outputNewLine="%CR;%LF;" binaryFloatRep="ieee"  textBidi="no" floating="no" 
+          textOutputMinLength="0">

```

Comment:
Add this property that Daffodil requires at the point of use where this format is referenced.

Daffodil probably needs to tolerate textStandardExponentCharacter (since this was edited to have the newer name textStandardExponentRep for daffodil.)

Reply:
Fixed this in Daffodil. So no longer need this customization of the IBM sample files.